web-app/src/InvoiceDocumentation/ubl2.1/rules/Rule.js

```diff
@@ -0,0 +1,43 @@
+import {useSelector} from 'react-redux'
+import {Card, Col, Row} from 'react-bootstrap'
+import {useTranslation} from 'react-i18next'
+import NotFound from '../../../helpers/NotFound'
+import {ubl21RulesDocsSelector} from '../../../cache/documentation/state'
+
+export default ({location}) => {
+  const {i18n, t} = useTranslation('common')
+
+  const ruleId = location.pathname.split('/')[3]
+  const docs = useSelector(ubl21RulesDocsSelector)
+
+  // Rule does not exist in invoice rules documentation
+  if (docs == null || !docs[ruleId]) return <NotFound />
```

Comment:
it should not happen that docs is `null`
let's create direct selector for current rule
```
export const ubl21RuleSelector = (id) => (state) => state.docs['ubl2.1'].rules.id
```
and use it
```
const docs = useSelector(ubl21RuleSelector(id))
```